cli/impl/src/test/java/io/helidon/build/cli/impl/TestUtils.java

```diff
@@ -142,6 +140,42 @@ static String execWithDirAndInput(File wd, File input, String... args) throws Ex
         return strip(output);
     }
 
+    /**
+     * Execute the CLI dev goal.
+     *
+     * @param wd    working directory
+     * @param args  command arguments
+     * @param environment  environment variables
+     * @return process monitor
+     * @throws Exception if an error occurs
+     */
+    static ProcessMonitor executeDevCommand(File wd, Map<String, String> environment, String... args) throws Exception {
```

Comment:
There is nothing 'dev is this method right ? It seems that the only difference between this new method and `execWithDirAndInput` is that it does not wait for completion and returns the `ProcessMonitor` instead of the output.

You can re-use the new method in the old one. Maybe call it `startWithDirAndInput` or something better.